tests/Query/BuilderTest.php

```diff
@@ -748,6 +753,48 @@ function (Builder $builder) {
             fn (Builder $builder) => $builder->where('name', 'like', '_ac__me_'),
         ];
 
+        yield 'whereLike' => [
+            ['find' => [['name' => new Regex('^1$', 'i')], []]],
+            fn
+        (Builder $builder) => $builder->whereLike('name', '1'),
```

Comment:
Nit: The indentation seems off in this data provider

Reply:
Phpcs does this 😥

Reply:
Oh wow. Didn't say a word then.